http/src/main/resources/jupyter/jupyter-docker-compose.yaml

```diff
@@ -37,3 +41,6 @@ services:
     env_file:
       - /etc/google_application_credentials.env
       - /etc/custom_env_vars.env
+    # See https://docs.docker.com/engine/reference/run/#user-memory-constraints
+    mem_limit: ${MEM_LIMIT}   # hard limit on memory consumption by the container
+    memswap_limit: -1         # no constraints on swap usage - use as much as allowed by the host
```

Comment:
We're not actually enabling swap on the VM yet, so this has no effect.

Reply:
can you document unit for this mem_limit variable

Reply:
Yeah… actually I may just remove it here because we're not enabling swap space anyway